app/views/job_templates/cloud_connector.erb

```diff
@@ -14,6 +14,12 @@ template_inputs:
   advanced: false
   value_type: plain
   hidden_value: true
+- name: http_proxy
```

Comment:
@adamruzicka, is it enough to add a parameter to the template to make it available to the underlying playbook?

Reply:
It should be, as long as the role looks for a variable with exactly the same name